explorer/src/lib/components/navbar/Navbar.svelte

```diff
@@ -56,11 +46,21 @@
     )[0]?.clientHeight;
   }
 
+  /**
+   * @param {Event} e
+   */
+  function handleChange(e) {
```

Comment:
I think `handleNetworkChange` would be a more fitting name, as we have many components here.